docs/openoni.md

```diff
@@ -141,10 +141,11 @@ For use with Nebraska Theme:
 `vim onisite/urls.py`:
 ```python
 from django.conf.urls import url, include
+from onisite.plugins.featured_content import views
 
 urlpatterns = [
   url(r'^calendar-', include("onisite.plugins.calendar.urls")),
-  url(r'^$', include("onisite.plugins.featured_content.urls")),
+  url(r'^$', views.featured, name="featured_home"),
```

Comment:
Ooooooh I missed that the views was coming from the featured_content plugin, hence my confusion.  I forgot that I should look at the top of the file to see where the variable names are coming from.  Not to be too nitpicky, but I was very confused that a generic "views" variable would be specific to the featured pages....what if it was something like

```
from onisite.plugins import featured_content

url(r'^$', featured_content.views.featured, name="featured_home"),
```
That's rather longer, but perhaps more clear.  Since this is Nebraska specific documentation, however, I can probably just deal with the more concise version if you prefer to keep it that way :)

Reply:
I know when I originally started work on this to clear up the warnings etc I tried it as:
```python
url(r'^$', onisite.plugins.featured_content.views.featured, name="featured_home"),
```

That resulted in an errors saying something like `onisite cannot be found`, so I think a `from onisite...` line of some sort is required.

When I added the code I thought, "This could be a problem if we need to import another `views` object... but we only have this one for now. I'll deal with that later...` But I'll try what you suggested as that seems less likely to cause conflicts later and reduces confusion with variable names.